rebuild-amplify/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
what is the master.txt? I dont think we need that?

Reply:
same for the other text files

Reply:
see description - i added the logs in as a safety measure. i can remove if you feel this is extra HAHA (cos we ran migration already)

Reply:
I don't think we should be committing the raw logs with the IDs etc.

Reply:
deleted